packages/web-components/src/components/cbp-usa-banner/cbp-usa-banner.scss

```diff
@@ -0,0 +1,66 @@
+/**
+ * @Prop   --cbp-usa-banner-background-color: var(--cbp-color-gray-cool-90);
+ * @Prop   --cbp-usa-banner-color: var(--cbp-color-text-lightest);
+ */
+ 
+ :root{
+  --cbp-usa-banner-background-color: var(--cbp-color-gray-cool-90);
+  --cbp-usa-banner-color: var(--cbp-color-text-lightest);
+}
+
+cbp-usa-banner {
+  background-color: var(--cbp-usa-banner-background-color);
+  color: var(--cbp-usa-banner-color);
+  display: inline-block;
+  width: 100%;
+  padding-block: var(--cbp-space-4x);
+  padding-inline: var(--cbp-space-8x);
+
+  --cbp-usa-banner-icon-rotate: rotate(90deg);
+
+  &:not([open]) .cbp-usa-banner-content {
+    display: none;
+  }
+
+  img{
+    display: inline;
+    margin-right: var(--cbp-space-3x);
+  }
+
+  .cbp-usa-banner-expand:not(#fakeId)  {
```

Comment:
this might not be needed if you use a button because the button's CSS API is more robust.

Reply:
replaced with link with button but the :not(#fakeId) was still needed for specificity